apps/windows/WindowsTests.js

```diff
@@ -39,27 +39,13 @@ export const windowsTests = [
     component: PressableTest,
     testPage: HOMEPAGE_PRESSABLE_BUTTON
   },
-  {
-    name: 'Link Test',
-    component: LinkTest,
-    testPage: HOMEPAGE_LINK_BUTTON
-  },
-  {
+  {name: 'Link Test', component: LinkTest, testPage: HOMEPAGE_LINK_BUTTON}, {
     name: 'Separator Test',
     component: SeparatorTest,
     testPage: HOMEPAGE_SEPARATOR_BUTTON
   },
-  {
-    name: 'Text Test',
-    component: TextTest,
-    testPage: HOMEPAGE_TEXT_BUTTON
-  },
-  {
-    name: 'Theme Test',
-    component: ThemeTest,
-    testPage: HOMEPAGE_THEME_BUTTON
-  },
-  {
+  {name: 'Text Test', component: TextTest, testPage: HOMEPAGE_TEXT_BUTTON},
+  {name: 'Theme Test', component: ThemeTest, testPage: HOMEPAGE_THEME_BUTTON}, {
```

Comment:
Nit, I'm assuming this this was an automated styling change? Is there anyway to revert the single line object? There's a mix between single-line and multi-line.

Reply:
Every time I try to reformat any TS file, the next build reformats it back according to some rules.   I've never worked in this repo before, so I'm not familiar with how the linting/formatting is tooled.